docs/embedded-entities.md

```diff
@@ -17,10 +17,10 @@ export class User {
     id: string;
     
     @Column()
-    nameFirst: string;
```

Comment:
I called them inverse for some reason, I dont remember why... something with final database column names

Reply:
Do you want me to revert these changes?